integration/spark/src/main/scala/org/apache/spark/sql/parser/CarbonSpark2SqlParser.scala

```diff
@@ -495,7 +495,9 @@ class CarbonSpark2SqlParser extends CarbonDDLSqlParser {
     (INTO ~> TABLE ~> (ident <~ ".").? ~ ident) ~
     (PARTITION ~> "(" ~> repsep(partitions, ",") <~ ")").? ~
     (OPTIONS ~> "(" ~> repsep(options, ",") <~ ")").? <~ opt(";") ^^ {
-      case filePath ~ isOverwrite ~ table ~ partitions ~ optionsList =>
+      case filePath ~ isOverwrite ~ table ~ partitions ~ optionsList
+        if CarbonPlanHelper.isCarbonTable(TableIdentifier(table._2, table._1),
```

Comment:
```suggestion
        if CarbonPlanHelper.isCarbonTable(TableIdentifier(table._2, table._1)
```
CarbonPlanHelper has isCarbonTable(tableIdent: TableIdentifier) method